App.tsx

```diff
@@ -40,11 +48,9 @@ export default function App() {
     const [draftCollectionLoc, setDraftCollectionLoc] = useState<DraftRequest | null>();
     const connect = useCallback(async () => {
         if (!client) {
-            let createdClient = await LogionClient.create({
-                directoryEndpoint: LOGION_DIRECTORY,
-                rpcEndpoints: [ LOGION_RPC ],
-                buildFileUploader: () => new ReactNativeFileUploader(),
-            });
+            const env = validEnvironmentOrThrow(LOGION_ENV);
+            const config = createLogionClientConfig(env);
+            let createdClient = await LogionClient.create(config);
```

Comment:
That did not hit me earlier but it would be nice to be able to directly pass the enum value (e.g. `Environment.DEV`) instead of having to do a manual lookup of the environment, then create the config object, then create the client. 

In other words, it would be nice to be able to write `let createdClient = await LogionClient.create(LOGION_ENV)` instead of the above 3 lines.

Reply:
OK good idea.